code/cryptography/affine_cipher/affine.c

```diff
@@ -0,0 +1,80 @@
+//affine.c
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍